documentation/Quickstart-on-how-to-write-tests.md

```diff
@@ -130,6 +142,76 @@ If we have existing recordings then the tests have been run against generated th
 > export TEST_MODE=playback
 > rushx test
 ```
+## How to push test recordings to assets repo
+We need to push test recording files to [asset repo](https://github.com/Azure/azure-sdk-assets) after testing your test cases.
+
+Before push your recording file, you must confirm that you are able to push recordings to the "azure-sdk-assets" repo, you need write-access to the assets repo.[Permissions to `Azure/azure-sdk-assets`](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60)
+
+### Push test recording
+
+#### New Package - No recorded tests
+This section assumes that your package is new to the JS repo and that you're trying to onboard your tests with recorder, and the asset-sync workflow.
+
+Generate an `sdk/<service-folder>/<package-name>/assets.json` file by running the following command.
+
+```bash
+npx dev-tool test-proxy init
+```
+Note: If you [install `dev-tool` globally](https://github.com/Azure/azure-sdk-for-js/tree/main/common/tools/dev-tool#installation), you don't need `npx` prefix in the above command
+
+This command would generate an assets.json file with an empty tag.
+
+Example `assets.json` with an empty tag:
+
+```
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "js",
+  "TagPrefix": "js/network/arm-network",
+  "Tag": ""
+}
+```
+
+Then go to the next step to [Existing package - Tests have been pushed before](#existing-package---tests-have-been-pushed-before).
+
+#### Existing package - Tests have been pushed before
```

Comment:
This guide is meant for the RLC packages, right?
Do you know how many existing packages for which recordings are present and have not been pushed to the assets repo?? If there are none, there is no point in adding this migration. Just "doing from scratch" workflow should be good.

Reply:
I think we could re-organize the sections to reduce confusion currently all existing packages(RLC or mgmt) are pushed to assent repo and the migration is taken from our team. But for customers they also have two scenarios:
1) the package is new and first time to run tests
2) the package's recordings are pushed to asset repo but customers are the first time to run test.